ci/check_history_consistency.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""
+Checks the consistency of the history, by comparing flask db
+output and the filesystem ordering of files.
+"""
+import os
+import re
+import subprocess
+
+
+def _get_db_revisions():
+    output = (
+        subprocess.check_output(['flask', 'db', 'history'])
+        .decode('utf-8')
+        .strip()
+        .split('\n')
+    )
+    pattern = re.compile(r'^.* -> ([a-zA-Z0-9]+)( \(head\))?,')
+    for line in output:
+        for obj in pattern.finditer(line):
+            yield obj.group(1)
+
+
+def _get_file_revisions():
```

Comment:
since you yield the revisions i'd rename it to `_iter_file_revisions()`

Reply:
Renamed